index.md

```diff
@@ -2,32 +2,17 @@
 
 layout: col-sidebar
 title: OWASP Bellingham
-tags: example-tag
+tags: bellingham whatcom
 region: North America
-meetup-group:
+meetup-group: OWASP-Bellingham
```

Comment:
Does this integrate with [Meetup - Bham OWASP](https://www.meetup.com/owasp-bellingham-chapter/) directly? Not sure how the config here works

Reply:
Yeah, it's supposed to link directly if the URL matches